docs/tigergraph.md

```diff
@@ -0,0 +1,16 @@
+# Quick launch TigerGraph and graph-app-kit
+This guide walks through setting up a TigerGraph database to configure with `graph-app-kit`.
+
+## Setup TigerGraph
+Head over to [TG Cloud](https://tgcloud.io/) and create an account
+Follow along to the steps in this video (it is slightly outdated so the pages may look different, but the steps are the same)
+[![https://raw.githubusercontent.com/akash-kaul/static_media/main/Screen%20Shot%202021-01-18%20at%203.20.04%20PM.png](https://www.youtube.com/watch?v=JARd9ULRP_I)]
+
+### Fraud Graph
+1. Click **Create Solution** and choose the **Fraud and Money Laundering Detection** starter kit.
```

Comment:
Specify these steps are in tg cloud?

Reply:
The youtube video , illustrates the steps to spin up a solution in TG Cloud